docs/admin-guide/admin-getting-started.md

```diff
@@ -109,10 +109,9 @@ image registry at <http://localhost:5000>.
 
 #### Prerequisites
 
-##### Install golang (1.17+)
+##### Install golang (1.19+)
 
-The zot project requires `golang 1.17` or newer. You can follow [these
-instructions](https://go.dev/learn/) to install the `golang` toolchain.
+The zot project requires `golang 1.19` or newer. You can follow the [golang instructions](https://go.dev/learn/) to install the `golang` toolchain.
```

Comment:
@rchincha is this version correct, we have `1.20` in the go.mod? 

Note: starting with go 1.21, the version in the go.mod will be strictly evaluated as the minimum supported version, and builds with older versions would fail.

Reply:
Changed to:

"Follow the [golang instructions](https://go.dev/learn/) to install the `golang` toolchain. After installation, make sure that the `path` environment variable or your IDE can find the toolchain.

> :pencil2: You must use a golang version of at least the minimum specified in [go.mod](https://github.com/project-zot/zot/go.mod) or the build will fail."